test/e2e/gateway_api_test.go

```diff
@@ -273,6 +279,18 @@ func ensureCRDs(t *testing.T) {
 	}
 }
 
+// ensureExperimentalCRDs tests that experimental Gateway API custom resource definitions exist.
+func ensureExperimentalCRDs(t *testing.T) {
+	t.Helper()
```

Comment:
You probable don't want the t.Helper() here either if we're getting rid of them to make it easier to debug/troubleshoot CI issues.

Reply:
I returned to [the previous idea of using `ensureExperimentalCRDs`](https://github.com/openshift/cluster-ingress-operator/pull/1200#discussion_r2005192321), so this function doesn't exist anymore.